rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -1180,15 +1193,23 @@ void gui_application::OnAppStateChanged(Qt::ApplicationState state)
 bool gui_application::native_event_filter::nativeEventFilter([[maybe_unused]] const QByteArray& eventType, [[maybe_unused]] void* message, [[maybe_unused]] qintptr* result)
 {
 #ifdef _WIN32
-	if (!Emu.IsRunning() && !g_raw_mouse_handler)
+	if (!Emu.IsRunning() && !Emu.IsStarting())
```

Comment:
Why this change?

Reply:
If a hotplug event happend during the starting phase, then it would get missed otherwise, since the hotplug thread starts quite early in the starting phase